test/libsolidity/smtCheckerTests/external_calls/call_with_value_1.sol

```diff
@@ -2,13 +2,17 @@ contract C {
 	function g(address payable i) public {
 		require(address(this).balance == 100);
 		i.call{value: 10}("");
-		// Disabled due to Spacer nondeterminism
-		//assert(address(this).balance == 90); // should hold
-		// Disabled due to Spacer nondeterminism
-		//assert(address(this).balance == 100); // should fail
+		assert(address(this).balance == 90); // should hold
+		assert(address(this).balance == 100); // should fail
 	}
 }
 // ====
 // SMTEngine: all
+// SMTIgnoreOS: macos
 // ----
 // Warning 9302: (96-117): Return value of low-level calls not used.
+// Warning 1218: (175-211): CHC: Error trying to invoke SMT solver.
```

Comment:
Is this okay?

Reply:
Yea those are just z3 errors when generating counterexamples